modules/apps/object/object-service/src/main/java/com/liferay/object/internal/field/business/type/PicklistObjectFieldBusinessType.java

```diff
@@ -158,13 +158,6 @@ public void predefineObjectFieldSettings(
 			List<ObjectFieldSetting> objectFieldSettings)
 		throws PortalException {
 
-		if (oldObjectField != null) {
-			_objectStateFlowLocalService.updateDefaultObjectStateFlow(
-				newObjectField, oldObjectField);
-
-			return;
-		}
```

Comment:
@G-Sa why did you remove this code? The `updateDefaultObjectStateFlow` method is only called here and I'm afraid of "discarding" all of its logic.

Reply:
@carolmariaabb That code meant that if the object field already existed it would only touch the "default" state flow, but wouldn't update the list in any way, so i moved the validation down below to do the whole update. 

But nice catch on the `updateDefaultObjectStateFlow`, its really missing, i will put it after the update of the whole list